modules/nw-route-specific-annotations.adoc

```diff
@@ -16,55 +16,122 @@ To create a whitelist with multiple source IPs or subnets, use a space-delimited
-|`router.openshift.io/haproxy.health.check.interval`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
+and "-". The default is the hashed internal key name for the route. 
+|
+|
+
+|`haproxy.router.openshift.io/pod-concurrent-connections`
+|Sets the maximum number of connections that are allowed to a backing pod from a router. +
+Note: If there are multiple pods, each can have this many connections.  If you have multiple routers, there is no coordination among them, each may connect this many times. If not set, or set to 0, there is no limit. 
+|
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections`
+|Setting `'true'` or `'TRUE'` enables rate limiting functionality which is implemented through stick-tables on the specific backend per route. +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks. 
+|
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections.concurrent-tcp`
+|Limits the number of concurrent TCP connections made through the same source IP address. It accepts a numeric value. +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+| 
+
+|`haproxy.router.openshift.io/rate-limit-connections.rate-http`
+|Limits the rate at which a client with the same source IP address can make HTTP requests. It accepts a numeric value.  +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections.rate-tcp`
+|Limits the rate at which a client with the same source IP address can make TCP connections. It accepts a numeric value.  +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+|
+
+|`haproxy.router.openshift.io/timeout` 
+|Sets a server-side timeout for the route. (TimeUnits) 
+|`spec.tuningOptions.serverTimeout` 
+|
+
+|`haproxy.router.openshift.io/timeout-tunnel` 
+|This timeout applies to a tunnel connection, for example, WebSocket over cleartext, edge, reencrypt, or passthrough routes. With cleartext, edge, or reencrypt route types, this annotation is applied as a timeout tunnel with the existing timeout value. For the passthrough route types, the annotation takes precedence over any existing timeout value set. 
+|`spec.tuningOptions.TunnelTimeout`
+|
+
+|`ingresses.config/cluster ingress.operator.openshift.io/hard-stop-after` 
+|You can set either an IngressController or the ingress config . This annotation redeploys the router and configures the HA proxy to emit the haproxy `hard-stop-after` global option, which defines the maximum time allowed to perform a clean soft-stop.
+|
+|`ingress.operator.openshift.io/hard-stop-after`
```

Comment:
I might have misunderstoood the original bug report, but I thought one of the requests was to remove this annotation (which is *not* a *route* annotation) from the table of route annotations, and perhaps add a new procedure to describe how to use hard-stop-after.

Reply:
@palonsoro I remember, we did have a discussion on this, whether to create a new section for hard-stop-after. Can you check again and let me know, if we have to create it.

Reply:
Let me remove the ingress controller field from the table and add a new section for hard-stop-after.